src/applications/pensions/tests/unit/chapters/05-financial-information/netWorthEstimation.unit.spec.jsx

```diff
@@ -67,11 +64,17 @@ describe('Financial information net worth estimation pension page', () => {
       </FakeProvider>,
     );
     expect($$('va-alert', container).length).to.equal(0);
-    const input = $('input[name="root_netWorthEstimation"]', container);
-    fireEvent.input(input, {
-      target: { name: 'root_netWorthEstimation', value: '25001' },
+
+    const webComponent = container.querySelector(
+      'va-text-input[name="root_netWorthEstimation"]',
+    );
+    webComponent.value = '25001';
```

Comment:
The currency pattern stores the value as a number. I just encountered a weird situation (in a different form) where I copy/pasted in the form data with string currency values and when I focus/blur the input without changing anything, an invalid type number error pops up for the input.

<img width="541" alt="Savings value stored as a string has an input error of 'is not of a type(s) number'" src="https://github.com/user-attachments/assets/bca02cc0-d32c-433c-84d6-51d296c939c3" />